storage/src/vespa/storage/distributor/distributorcomponent.cpp

```diff
@@ -281,6 +281,11 @@ DistributorComponent::updateBucketDatabase(
     if (updateFlags & DatabaseUpdate::RESET_TRUSTED) {
         dbentry->resetTrusted();
     }
+    if (dbentry->getNodeCount() == 0) {
+        LOG(warning, "all nodes in changedNodes set (size %zu) are down, removing dbentry", changedNodes.size());
```

Comment:
Since this now known to be able to happen during normal operation, I suggest lowering this to debug instead of warning. Unless we want to leave it like this for a while for the sake of visibility.